app/kubernetes/components/datatables/applications-datatable/applicationsDatatableController.js

```diff
@@ -25,13 +85,29 @@ angular.module('portainer.docker').controller('KubernetesApplicationsDatatableCo
     };
 
     this.isSystemNamespace = function (item) {
+      // if all charts in a helm app/release are in the system namespace
+      if (item.KubernetesApplications && item.KubernetesApplications.length > 0) {
```

Comment:
the property `item.KubernetesApplications` will not exist on kubernetes applications - but will exist in helm applications (since a helm app encapsulates k8s apps).
In this case we need to check if the list/property exists and if there is an item inside it.

Reply:
right 👍